src/main/java/uk/gov/hmcts/darts/event/mapper/EventMapper.java

```diff
@@ -3,49 +3,86 @@
+        EventMapping eventMappingDetails = new EventMapping();
         eventMappingDetails.setId(eventEntity.getEventType().getId());
-        adminGetEventsForIdResponse.setEventMapping(eventMappingDetails);
+        eventMappingDetails.setName(eventEntity.getEventType().getEventName());
+        adminGetEventResponseDetail.setEventMapping(eventMappingDetails);
 
         CourthouseResponseDetails courthouseResponseDetails = new CourthouseResponseDetails();
         courthouseResponseDetails.setId(eventEntity.getCourtroom().getCourthouse().getId());
         courthouseResponseDetails.setDisplayName(eventEntity.getCourtroom().getCourthouse().getDisplayName());
-        adminGetEventsForIdResponse.setCourthouse(courthouseResponseDetails);
+        adminGetEventResponseDetail.setCourthouse(courthouseResponseDetails);
 
         CourtroomResponseDetails courtroomResponseDetails = new CourtroomResponseDetails();
         courtroomResponseDetails.setId(eventEntity.getCourtroom().getId());
         courtroomResponseDetails.setName(eventEntity.getCourtroom().getName());
-        adminGetEventsForIdResponse.setCourtroom(courtroomResponseDetails);
-
-        adminGetEventsForIdResponse.setVersion(eventEntity.getLegacyVersionLabel());
-        adminGetEventsForIdResponse.setChronicleId(eventEntity.getChronicleId());
-        adminGetEventsForIdResponse.setAntecedentId(eventEntity.getAntecedentId());
-        adminGetEventsForIdResponse.setEventTs(eventEntity.getTimestamp());
-        adminGetEventsForIdResponse.isCurrent(eventEntity.getIsCurrent());
-        adminGetEventsForIdResponse.setCreatedAt(eventEntity.getCreatedDateTime());
-        adminGetEventsForIdResponse.setCreatedBy(eventEntity.getCreatedBy().getId());
-        adminGetEventsForIdResponse.setLastModifiedAt(eventEntity.getLastModifiedDateTime());
-        adminGetEventsForIdResponse.setLastModifiedBy(eventEntity.getLastModifiedBy().getId());
-        adminGetEventsForIdResponse.setIsDataAnonymised(eventEntity.isDataAnonymised());
+        adminGetEventResponseDetail.setCourtroom(courtroomResponseDetails);
+
+        adminGetEventResponseDetail.setVersion(eventEntity.getLegacyVersionLabel());
+        adminGetEventResponseDetail.setChronicleId(eventEntity.getChronicleId());
+        adminGetEventResponseDetail.setAntecedentId(eventEntity.getAntecedentId());
+        adminGetEventResponseDetail.setEventTs(eventEntity.getTimestamp());
+        adminGetEventResponseDetail.isCurrent(eventEntity.getIsCurrent());
+        adminGetEventResponseDetail.setCreatedAt(eventEntity.getCreatedDateTime());
+        adminGetEventResponseDetail.setCreatedBy(eventEntity.getCreatedBy().getId());
+        adminGetEventResponseDetail.setLastModifiedAt(eventEntity.getLastModifiedDateTime());
+        adminGetEventResponseDetail.setLastModifiedBy(eventEntity.getLastModifiedBy().getId());
+        adminGetEventResponseDetail.setIsDataAnonymised(eventEntity.isDataAnonymised());
```

Comment:
A bunch of these don't align with what the LLD says

Reply:
This object is reused from another endpoint. I've updated the LLD to add the few extra fields in the response.